src/patternfly/demos/Table/table-expanded-table.hbs

```diff
@@ -1,7 +1,7 @@
 {{#> table table--id=(concat page--id '-table') table--grid="true" table--modifier="pf-m-grid-xl" table--expandable="true" table--attribute='aria-label="Expanded table example"'}}
```

Comment:
You should be able to remove this, too. Looks to only be used in the "Toolbar expand all (expanded)" toolbar demo removed in this PR.